packages/scrollbars/src/Scrollbar.js

```diff
@@ -0,0 +1,165 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import styled from 'styled-components';
+import PerfectScrollbar from 'perfect-scrollbar';
+import { isRtl, retrieveTheme } from '@zendeskgarden/react-theming';
```

Comment:
Horizontal scrolling for (not so)PerfectScrollbar is completely broken for RTL. Vertical isn't great, either.

Reply:
Yeah I'm seeing some weird behaviour with rtl mode on